### PR TITLE
Updating HCA/BRAIN pipeline overviews and adding RRIDs

### PR DESCRIPTION
**Pull request by ekiernan**
* Updated Overviews and Methods Docs to include citation sections and SciCrunch RRIDs. 
* Fixed typos and broken links
* Updated language

### CODE REVIEW

File: website/docs/documentation/Pipelines/Optimus_Pipeline/README.md [hunk 1]
```diff
@@ -211,21 +211,25 @@ Please note that we have deprecated the previously used Zarr array output. The p
...
 
- 
 
 ## Versioning
 
 All Optimus pipeline releases are documented in the [Optimus changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/optimus/Optimus.changelog.md).
 
+
+## Citing the Optimus Pipeline
+Please identity the pipeline in your methods section using the Optimus Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018908/resolver?q=SCR_018908&l=SCR_018908).
```
Comment (kbergin) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/README.md:230]:
```suggestion
Please identify the pipeline in your methods section using the Optimus Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018908/resolver?q=SCR_018908&l=SCR_018908).
```

File: website/docs/documentation/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md [hunk 2]
```diff
@@ -151,6 +151,10 @@ The following table details the metrics available in the output_snap_qc file.
 
 All scATAC workflow releases are documented in the [scATAC changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/scATAC/scATAC.changelog.md).
 
+## Citing the scATAC Pipeline
+Please identity the pipeline in your methods section using the scATAC Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018919/resolver?q=SCR_018919&l=SCR_018919).
```
Comment (kbergin) [inline website/docs/documentation/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md:155]:
```suggestion
Please identify the pipeline in your methods section using the scATAC Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018919/resolver?q=SCR_018919&l=SCR_018919).
```

File: website/docs/documentation/Pipelines/Smart-seq2_Multi_Sample_Pipeline/README.md [hunk 3]
```diff
@@ -93,4 +91,14 @@ The Smart-seq2 Multi Sample pipeline has been validated for processing human and
...
+Release information for the Multi-SS2 Pipeline can be found in the [changelog](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md). Please note that any major changes to the Smart-seq2 pipeline will be documented in the [Smart-seq2 Single Sample changelog](https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md).
+
+## Citing the Smart-seq2 Multi-Sample Pipeline
+Please identity the pipeline in your methods section using the Smart-seq2 Multi-Sample Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018920/resolver?q=Smart-seq2&l=Smart-seq2).
```
Comment (kbergin) [inline website/docs/documentation/Pipelines/Smart-seq2_Multi_Sample_Pipeline/README.md:97]:
```suggestion
Please identify the pipeline in your methods section using the Smart-seq2 Multi-Sample Pipeline's [SciCrunch resource identifier](https://scicrunch.org/scicrunch/Resources/record/nlx_144509-1/SCR_018920/resolver?q=Smart-seq2&l=Smart-seq2).
```

File: website/docs/documentation/Pipelines/Smart-seq2_Single_Sample_Pipeline/README.md [hunk 4]
```diff
@@ -203,7 +203,12 @@ The Smart-seq2 Single Sample pipeline has been validated for processing human an
 | Human stranded fluidigm | [Report](https://docs.google.com/document/d/1FEg86Tlu657j9Kjw_v3keFQRXcBIs8gOqCwLbPSP-C0/edit#heading=h.wjr8otl7zg14) | 
 
 ## Versioning
-All Smart-seq2 Single Sample release notes are documented in the [Smartseq2 Single Sample changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md).
+All SS2 release notes are documented in the [Smartseq2 Single Sample changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md).
+
+## ## Citing the Smart-seq2 Single Sample Pipeline
+Please identity the SS2 pipeline in your methods section using the Smart-seq2 Single Sample Pipeline's [SciCrunch resource identifier](https://scicrunch.org/browse/resourcedashboard).
```
Comment (kbergin) [inline website/docs/documentation/Pipelines/Smart-seq2_Single_Sample_Pipeline/README.md:209]:
```suggestion
Please identify the SS2 pipeline in your methods section using the Smart-seq2 Single Sample Pipeline's [SciCrunch resource identifier](https://scicrunch.org/browse/resourcedashboard).
```

**Review by kbergin — APPROVED** [review]

File: website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+Below we provide a sample methods sections for a publication, separated into single-cell or single-nucleus use cases. For the complete pipeline documentation, see the [Optimus Overview](./README.md).
 
 # Methods
 
 ## Single-cell (sc_rna mode)
-Data preprocessing and count matrix construction were performed using the Optimus v4.1.7 Pipeline. Briefly, FASTQ files were converted to unaligned BAM (uBMA) using Picard v2.10.10 and reads were appended with raw UMI and corrected cell barcode sequences using Single Cell Tools (sctools) v0.3.12 and the 10x Genomics barcodes whitelist, allowing for up to one edit distance (Levenshtein distance).
+Data preprocessing and count matrix construction were performed using the Optimus v4.2.4 Pipeline (RRID:SCR_018908). Briefly, FASTQ files were converted to unaligned BAM (uBMA) using Picard v2.10.10 and reads were appended with raw UMI and corrected cell barcode sequences using Single Cell Tools (sctools) v0.3.12 and the 10x Genomics barcodes whitelist, allowing for up to one edit distance (Levenshtein distance).
```
Comment (nikellepetrillo) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md:8]: Should uBMA be uBAM?

File: website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md [hunk 6]
```diff
@@ -34,9 +34,9 @@ All cell and gene metrics (alignment, mitochondrial, and other QC metrics), coun
...
 
-## Single-nuclei (sn_rna mode)
+## Single-nucleus (sn_rna mode)
 
-Data preprocessing and count matrix construction were performed using the Optimus v4.1.7 Pipeline. Briefly, FASTQ files were converted to unaligned BAM (uBMA) using Picard v2.10.10 and reads were appended with raw UMI and corrected cell barcode sequences using Single Cell Tools (sctools) v0.3.11 and the 10x Genomics barcodes whitelist, allowing for up to one edit distance (Levenshtein distance).
+Data preprocessing and count matrix construction were performed using the Optimus v4.1.7 Pipeline (RRID:SCR_018908). Briefly, FASTQ files were converted to unaligned BAM (uBMA) using Picard v2.10.10 and reads were appended with raw UMI and corrected cell barcode sequences using Single Cell Tools (sctools) v0.3.11 and the 10x Genomics barcodes whitelist, allowing for up to one edit distance (Levenshtein distance).
```
Comment (nikellepetrillo) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md:39]: same questions, should uBMA be uBAM?

**Review by nikellepetrillo — APPROVED** [review]
Looks good to me, just 2 comments

File: website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md (hunk 5 above)
Comment (ekiernan) [reply website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md]: Yes- thanks for catching!

File: website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md (hunk 6 above)
Comment (ekiernan) [reply website/docs/documentation/Pipelines/Optimus_Pipeline/optimus.methods.md]: Yes, thanks for catching!

**Pull request merged**